cmd/juno/juno.go

```diff
@@ -174,8 +174,11 @@ const (
 	dbCacheSizeUsage     = "Determines the amount of memory (in megabytes) allocated for caching data in the database."
 	dbMaxHandlesUsage    = "A soft limit on the number of open files that can be used by the DB"
 	gwAPIKeyUsage        = "API key for gateway endpoints to avoid throttling" //nolint: gosec
-	gwTimeoutUsage       = "Timeout for requests made to the gateway"          //nolint: gosec
-	callMaxStepsUsage    = "Maximum number of steps to be executed in starknet_call requests. " +
+	gwTimeoutsUsage      = "Timeouts for requests made to the gateway. Can be specified in three ways:\n" +
+		"- Single value (e.g. '5s'): After each failure, the timeout will increase exponentially\n" +
```

Comment:
It won't increase exponentially 😨 , it will increased dynamically, based on a heuristic but not exponentially.

Nonetheless, great explanation!